index.html

```diff
@@ -0,0 +1,259 @@
+            <img src="./images/John-Derera.jpg" alt="" class="speaker-img">
+          </div>
+          <div class="speaker-details">
+            <h4 class="speaker-name">John Derera</h4>
+            <p class="speaker-title"><em>Head of Breeding IITA</em></p>
+            <div class="speaker-line"></div>
+            <p class="speaker-text">John Derera, a Zimbabwean, is currently the Head of Breeding in IITA. He is a
+              specialist in Maize Breeding, Quantitative & Biometrical Genetics and R&D.</p>
+          </div>
+        </div>
+      </div>
+
+      <div class="test">
+        <div id="card5">
+          <div class="dot-photo">
+            <img src="./images/rectdot.png" alt="" class="dot">
+            <img src="./images/Richard-Keith-Downey.jpeg" alt="" class="speaker-img">
+          </div>
+          <div class="speaker-details">
+            <h4 class="speaker-name">Richard Keith Downey</h4>
+            <p class="speaker-title"><em>Canadian Agricultural Scientist</em></p>
+            <div class="speaker-line"></div>
+            <p class="speaker-text">Keith Downey, known as the “Father of CANOLA,” is a plant breeder who was largely
+              responsible for transforming rapeseed into canola. This crop, first grown on the PRAIRIES during WORLD WAR
+              II for industrial oil.</p>
+          </div>
+        </div>
+      </div>
+
+          <div class="test">
+          <div id="card6">
+            <div class="dot-photo">
+              <img src="./images/rectdot.png" alt="" class="dot">
+              <img src="./images/monty_jones.jpg" alt="" class="speaker-img">
+            </div>
+            <div class="speaker-details">
+              <h4 class="speaker-name">Monty Jones</h4>
+              <p class="speaker-title"><em>Agricultural & Nutritional Scientist</em></p>
+              <div class="speaker-line"></div>
+              <p class="speaker-text">He began his professional research career at the National Rice Research Station at
+                Rokupr in Sierra Leone in 1975 where he pioneered research in rice breeding and rice varietal
+                improvement program.
+              </p>
+            </div>
+          </div>
+        </div> -->
+        </div>
+    </section>
+
+
+
+
+
+    <section class="partners">
+      <h4 class="partners-header">Partners</h4>
+      <div class="h_line"></div>
+      <div class="partners-image">
+        <img src="./images/partner-img1.jpeg" alt="agrol-mall">
+        <img src="./images/partner-img2.png" alt="farm-kart">
+        <img src="./images/partner-img3.jpg" alt="tiger-brands">
+        <img src="./images/partner-img4.png" alt="cevital">
+        <img src="./images/partner-img5.png" alt="danone">
+      </div>
+    </section>
+  </main>
+  <footer>
+    <img src="./images/logo.png" alt="logo" class="footer-logo">
+    <div class="footer-text">
+      <p class="footer-date">2023 Lagos|Ngeria</p>
```

Comment:
- [ ] Dear @oluwatobi-fasasi, For your footer please notice the footer is white on the mobile design. Also please notice your footer is not the same as the original design. As a front-end engineer, we need to make sure we mimic the given design and your design could use some arrangement. 

### Your mobile footer
![image](https://user-images.githubusercontent.com/69077061/227595632-c8b5c5ca-0a2c-4d78-a7bc-a290bef7debd.png)
### Original design footer
![image](https://user-images.githubusercontent.com/69077061/227595866-e00b6d93-cca2-40a0-b49c-a4dde6e4c1ae.png)
---
- [ ] On the desktop display the footer of the about page has a dark background and on the home page, It has a white background, please notice you have white on both of the pages. Please try to address the difference and make some amendments. 

### Your desktop footer on both pages
![image](https://user-images.githubusercontent.com/69077061/227596798-e7c23504-7ec9-468b-88fa-6f0fc79d40b7.png)
### Orginal design footer for home page 
![image](https://user-images.githubusercontent.com/69077061/227596990-bd713d38-06e3-482d-a54a-d311ab92394e.png)
### Orginal design footer for about page 
![image](https://user-images.githubusercontent.com/69077061/227597050-df4ba0cf-72bc-43d4-86f1-08f9fc1b1dab.png)